public/js/src/game/HanabiCard.js

```diff
@@ -126,13 +126,13 @@ class HanabiCard extends Phaser.GameObjects.Container {
         const suit = (!this.showOnlyLearned && this.trueSuit);
         const empathyPastSuitUncertain = this.showOnlyLearned && this.possibleSuits.length > 1;
 
-        let suitToShow = suit || learnedCard.suit || SUIT.GRAY;
+        let suitToShow = suit || learnedCard.suit || SUIT.GRAY_UNKNOWN;
         if (empathyPastSuitUncertain) {
-            suitToShow = SUIT.GRAY;
+            suitToShow = SUIT.GRAY_UNKNOWN;
         }
 
-        // "Card-Gray" is not created, so use "NoPip-Gray"
-        if (suitToShow === SUIT.GRAY) {
+        // "Card-GRAY_UNKNOWN" is not created, so use "NoPip-GRAY_UNKNOWN"
```

Comment:
you should look at your commits before submitting a pull request and you would see errors like this